.github/workflows/ci.yml

```
@@ -644,20 +648,21 @@ jobs:
         run: ls -R
 
       - name: Upload to Private PyPi
-        run: |
-          pip install twine
-          python -m twine upload --skip-existing ./**/*.whl
-          python -m twine upload --skip-existing ./**/*.tar.gz
+        uses: pypa/gh-action-pypi-publish@v1.12.4
+        with:
+          packages-dir: dist
+          repository-url: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload
+          skip-existing: true
```

```suggestion
        run: |
          pip install twine
          python -m twine upload --skip-existing ./**/*.whl
          python -m twine upload --skip-existing ./**/*.tar.gz
```

The private PyPI needs to use the token approach